samples/azure-quantum/resource-estimation/estimation-time-qubits-constraints.ipynb

```diff
@@ -0,0 +1,486 @@
+    "        params.items[i].qec_scheme.name = single_shot_params.items[param_index].qec_scheme.name\n",
+    "        max_duration *= multiplier\n",
+    "\n",
+    "job = estimator.submit(IsingModel2DSim, input_params=params)\n",
+    "job.wait_until_completed(timeout_secs=300)\n",
+    "results_duration = job.get_results()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Visualize resutls."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plot_results(results_duration, labels, num_shots, 'Ising model estimates with duration constraints')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Experiments with `max_physical_qubits` Constraint\n",
+    "\n",
+    "\n",
+    "We will perform experiments by gradually reducing the `max_physical_qubits` constraint. In contrast to the previous `max_duration` constraint experiments, we will take the opposite approach by progressively tightening the constraint from shot to shot."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "multiplier = 1.05\n",
+    "num_shots = 100\n",
+    "\n",
+    "params = estimator.make_params(num_items=6 * num_shots)\n",
+    "\n",
+    "params.arguments = single_shot_params.arguments\n",
+    "\n",
+    "for param_index in range(6):\n",
+    "    max_physical_qubits = single_shot_y[param_index]\n",
+    "    for shot_index in range(num_shots):   \n",
+    "        i = param_index * num_shots + shot_index\n",
+    "        params.items[i].constraints.max_physical_qubits = int(max_physical_qubits)\n",
+    "        params.items[i].qubit_params.name = single_shot_params.items[param_index].qubit_params.name\n",
+    "        params.items[i].qec_scheme.name = single_shot_params.items[param_index].qec_scheme.name\n",
+    "        max_physical_qubits /= multiplier\n",
+    "\n",
+    "job = estimator.submit(IsingModel2DSim, input_params=params)\n",
+    "job.wait_until_completed(timeout_secs=300)\n",
+    "\n",
+    "results_qubits = job.get_results()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Visualize results."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plot_results(results_qubits, labels, num_shots, 'Ising model estimates with qubit constraints')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Next steps\n",
+    "\n",
+    "As we progress, consider exploring alternative algorithms for estimation. The algorithm used for estimation doesn't necessarily have to be an embedded Q# algorithm; you have the flexibility to leverage a variety of examples from the [Quantum GitHub repository](README.md).\n",
```

Comment:
I am not sure whether we should use absolute URIs here for when these notebooks are embedded in Azure Quantum notebooks.